mitmproxy/web/app.py

```diff
@@ -336,7 +369,21 @@ def get(self, flow_id, message):
         self.set_header("Content-Type", "application/text")
         self.set_header("X-Content-Type-Options", "nosniff")
         self.set_header("X-Frame-Options", "DENY")
-        self.write(message.raw_content)
+
+        cv = self.get_argument("cv", None)
+        if cv:
+            self.set_header("Content-Encoding", "")
+            viewmode = next(v for v in contentviews.views if v.name == cv)
+            description, lines  = self._get_content_view(
+                 message, viewmode
+            )
+
+            self.write(dict(
```

Comment:
This may need a JSON content type - at this point, I believe it makes sense to move the normal content and contentviews into two distinct endpoints.